library/decls.ml

```diff
@@ -18,7 +18,7 @@ open Libnames
 (** Datas associated to section variables and local definitions *)
 
 type variable_data =
-  DirPath.t * bool (* opacity *) * Univ.ContextSet.t * polymorphic * logical_kind
+  DirPath.t * bool (* opacity *) * Univ.ContextSet.t * bool (* poly *) * logical_kind
```

Comment:
Maybe make this a record?

Reply:
See #10417 , in fact I'd like to get rid of `Decls` and `Decl_kinds` altogether as it is only used for Dumpglob [I mean, moving this info to Dumpglob], so in this case I'd rather see if I can do a PR removing that, running CI on it now.